File: techhalo/src/app/common/Supplier.ts

export class Supplier {
    'supplierId': number;
    'name': string;
    'image': string;
    'address': string;
    'email': string;
    'phone': string;
    'create_at': Date;

    constructor(id: number, name: string) {
        this.supplierId = id;
        this.name = name;
    }
}

File: techhalo_Backend/src/main/java/vn/fs/entity/Supplier.java
package vn.fs.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.io.Serializable;
import java.time.LocalDate;

@Data
@Entity
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "suppliers")
public class Supplier implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long supplierId;
    private String image;
    private String name;
    private String email;
    private String phone;
    private String address;
    private LocalDate create_at;

}
